draft-ietf-httpbis-compression-dictionary.md

```diff
@@ -340,17 +377,22 @@ Vary: accept-encoding, available-dictionary
 
 ## Content Encoding
 
-IANA is asked to enter the following into the "HTTP Content Coding Registry" registry ({{HTTP}}):
+IANA is asked to enter the following into the "HTTP Content Coding Registry"
+registry ({{HTTP}}):
 
 - Name: br-d
-- Description: A stream of bytes compressed using the Brotli protocol with an external dictionary of not more than 16 MB.
+- Description: A stream of bytes compressed using the Brotli protocol with an
+external dictionary of not more than 16 MB.
 - Reference: This document
 - Notes: {{compression-algorithms}}
 
-IANA is asked to enter the following into the "HTTP Content Coding Registry" registry ({{HTTP}}):
+IANA is asked to enter the following into the "HTTP Content Coding Registry"
+registry ({{HTTP}}):
 
 - Name: zstd-d
-- Description: A stream of bytes compressed using the Zstandard protocol with an external dictionary of not more than 8 MB.
+- Description: A stream of bytes compressed using the Zstandard protocol with
+an external dictionary of not more than the larger of 8 MB and the size of the
+dictionary.
```

Comment:
If we want to keep the discussion of the window size here, I would propose phrasing this as:

> A stream of bytes compressed using the Zstandard protocol using a negotiated external dictionary. Unlike the `zstd` Content Coding, the Zstandard frame(s) making up the stream may use a `Window_Size` up to and including the size of the external dictionary or 8 MB, whichever is larger.

This makes it clear that the constraint is on the window, and not the size of the dictionary: "an external dictionary of not more than [...] the size of the dictionary" is tautological.

If we take @Cyan4973's suggestion in [this comment](https://github.com/httpwg/http-extensions/issues/2754#issuecomment-2061998009) (and I agree wiggle room would be nice to have here) I guess we'll have to complicate this a little further.

But since you are actually specifying this above, maybe it doesn't even need to be mentioned here.

Reply:
The main reason to mention it here is that this chunk of text is what is going to be in the IANA registry for the content encodings.  I fixed the language around it being the window size and not the dictionary, thanks.